src/main/java/Kafka.java

```diff
@@ -0,0 +1,136 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Kafka {
+
+    public ArrayList<Task> tasks = new ArrayList<>();
+
+    public void greet() {
+        String message = """
+              Hello. Kafka here.
+              We meet again.
+            """;
+        System.out.println(message);
+    }
+
+    public void goodbye() {
+        String message = "  Farewell. I look forward to our next meeting, wherever destiny may lead us. ";
+        System.out.println(message);
+    }
+
+    public void addTask(Task task) {
+        this.tasks.add(task);
+        System.out.println("  Got it. I've added this task.");
+        System.out.println("    " + task);
+        System.out.println("  Now you have " + this.tasks.size() + " task(s) in the list.");
+    }
+
+    public void createList() {
```

Comment:
The function name is a little misleading as it does not create a new list. Perhaps getList or printList might be better?